code/modules/virus2/analyser.dm

```diff
@@ -94,7 +94,7 @@
 
 /obj/machinery/disease2/diseaseanalyser/Topic(href, href_list)
 	if(..())
-		return 1
+		return
```

Comment:
I always forget one thing or another, I'm gonna make that the same across all 4 machines and amend the commit.